src/utils/embed-handler.js

```diff
@@ -20,7 +20,7 @@ function embed({ EMBED_SERVICE } = {}) {
     const { url } = node;
     let src = url;
     if (URI.parse(url).reference === 'absolute') {
-      src = URI.normalize([EMBED_SERVICE, url.slice(-1) === '/' ? '/' : '', url].join(''));
+      src = URI.normalize([EMBED_SERVICE, EMBED_SERVICE.slice(-1) !== '/' ? '/' : '', url].join(''));
```

Comment:
why not formulate it positive? 

```suggestion
      src = URI.normalize([EMBED_SERVICE, EMBED_SERVICE.slice(-1) === '/' ? '' : '/'', url].join(''));
```

Reply:
It feels more logical to me to read: `if there is no slash, add one, otherwise do nothing` than `if there is a slash, do nothing, otherwise add one`...